mdtraj/core/trajectory.py

```diff
@@ -1669,6 +1669,80 @@ def remove_solvent(self, exclude=None, inplace=False):
 
         return self.atom_slice(atom_indices, inplace = inplace)
 
+    def smooth(self, width, order=3, atom_indices=None, inplace=False):
+        """
```

Comment:
No need for the extra linebreak here, and put the reference to the scipy cookbook as a citation instead (e.g. https://github.com/mdtraj/mdtraj/blob/master/mdtraj/geometry/dssp.py#L75-L78).